foundryup/install

```diff
@@ -73,34 +72,31 @@ else
   fi
 
   REPO_PATH="${FOUNDRY_DIR}/${FOUNDRYUP_REPO}"
-
-  # Delete any previous installations if they exist as they will now be using one from cargo and we dont want conflicts
-  rm -f $FOUNDRY_BIN_DIR/forge
-  rm -f $FOUNDRY_BIN_DIR/cast
-
+  
   if [ -d $REPO_PATH ]; then
-    # If the repo path exists move to it, git pull, and checkout the branch and cargo install local.
+    # If the repo path exists move to it git pull and checkout the branch.
     cd $REPO_PATH
     git pull
     git checkout ${FOUNDRYUP_BRANCH}
     git pull
-    cargo install --path ./cli --bins --locked --force
   else
-    # Repo path did not exist, grab the author from the repo, make a directory in .foundry, cd to it, clone and install.
+    # Repo path did not exist, grab the author from the repo, make a directory in .foundry, cd to it and clone.
     IFS="/" read -ra AUTHOR <<< "$FOUNDRYUP_REPO"
     mkdir -p "$FOUNDRY_DIR/$AUTHOR"
     cd "$FOUNDRY_DIR/$AUTHOR"
     git clone https://github.com/${FOUNDRYUP_REPO}
     cd $REPO_PATH
     git checkout ${FOUNDRYUP_BRANCH}
-
-    cargo install --path ./cli --bins --locked --force --root $FOUNDRY_BIN_DIR
   fi
+  
+  # Build the repo and install it locally to the .foundry bin directory.
+  # --root tells cargo to put the compiled binaries in $FOUNDRY_DIR/bin!
+  cargo install --path ./cli --bins --locked --force --root $FOUNDRY_DIR
```

Comment:
Will it actually end up in `$FOUNDRY_DIR/bin` or in `$FOUNDRY_DIR`?

Reply:
yeah its confusing which is why i left that comment (ig its still confusing lol)

--root appends /bin to whatevers given to it, so we have to give it FOUNDRY_DIR not FOUNDRY_BIN_DIR or it will put it in .foundry/bin/bin